CSharp/ArticleProjects/EfficientListQueries/EfficientListQueries.csproj

```diff
@@ -36,8 +36,8 @@
       <HintPath>packages\Hyak.Common.1.0.2\lib\net45\Hyak.Common.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Batch, Version=2.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>packages\Azure.Batch.2.0.2\lib\net45\Microsoft.Azure.Batch.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Batch, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>packages\Azure.Batch.3.0.0\lib\net45\Microsoft.Azure.Batch.dll</HintPath>

```

Comment:
Charlie already made these changes in the sample projects, he was just dealing with a Travis error before merging: https://github.com/Azure/azure-batch-samples/pull/92

You might want to just wait for his to go through, then resubmit with only the Batch Explorer changes.